src/index.html

```diff
@@ -10,8 +10,206 @@
+              name="terms"
+              id="terms"
+              required
+            >
+          </label>
+        </div>
+      </fieldset>
+
+      <fieldset>
+        <legend>Registration</legend>
+        <div class="input">
+          <label>
+            E-mail:
+            <input
+              type="email"
+              name="email"
+              id="email"
+              placeholder="email@example.com"
+              required
+            >
+          </label>
+        </div>
+        <div class="input">
+          <label>
+            Password:
+            <input
+              type="password"
+              name="password"
+              id="password"
+              required
+            >
+          </label>
+        </div>
+      </fieldset>
+
+      <fieldset>
+        <legend>An interesting fact about you!</legend>
+        <div class="input">
+          <label>
+            Do you love cats?
+            <label>
+              <input
+                type="radio"
+                name="cats"
+                value="yes"
+              >
+              Yes
+            </label>
+            <label>
+              <input
+                type="radio"
+                name="cats"
+                value="no"
+              >
+              No
+            </label>
+          </label>
+        </div>
+        <div class="input">
+          <label>
+            What is your favorite color?
+            <input
+              type="color"
+              name="fvColor"
+            >
+          </label>
+        </div>
+        <div class="input">
+          <label>
+            What time do you go to bed?
+            <input
+              type="time"
+              name="bedTime"
+            >
+          </label>
+        </div>
+        <div class="input">
+          <label>
+            What are you favorite brand of cars?
+            <select 
+              name="carBrand"
+              multiple 
+            >
+              <option value="BMW">BMW</option>
+              <option value="Audi">Audi</option>
+              <option value="Lada">Lada</option>
+            </select>
+          </label>
+        </div>
+        <div class="input">
+          <label>
+            How do you rate our work?
+            <input
+              type="range"
+              name="rateWork"
+              value="0"
+            >
+          </label>
+        </div>
+      </fieldset>
+
+      <fieldset>
+        <legend>Additional info:</legend>
+        <div class="input">
+          <label>
+            Comments:
+            <textarea
+              
+              name="comments"
+              id="commets"
+              minlength="0"
+              maxlength="50"
+              autocomplete="off"
+           ></textarea>
+          </label>
+        </div>
+        <div class="input">
+          <label>
+            Would you recommend us?
+            <select
+              name="rec"
+              id="rec"
```

Comment:
let's not use shortcuts/abbreviations. You want your code to be maximally clear for those who read it